src/IndicatorCard.jsx

```diff
@@ -32,17 +32,17 @@ class IndicatorCard extends React.Component {
 
   render() {
     const {
-      areaName, code, size, name, values, indicatorIds,
+      activeArea, code, size, name, values, indicatorIds,
     } = this.props;
     const indicatorIdsQuery = indicatorIds.map((ind) => `ids=${ind}`).join('&');
     const className = this.validClassIndicator(size).validClass;
 
     if (!values) return null;
-    const { results, groups } = GraphData.prepareData(code, values);
+    const { results, groups } = GraphData.prepareData(code, values, 0, activeArea);
     return (
       <Link
         to={{
-          pathname: `/indicator/${areaName}`,
+          pathname: `/indicator/${activeArea.name}`,
```

Comment:
You should keep areaName instead of activeArea.name because of the error when reloading the dash component